src/client/utils/dataset/dataset.mocha.ts

```diff
@@ -18,43 +18,46 @@
 import { expect } from "chai";
 import { Timezone } from "chronoshift";
 import { Dataset } from "plywood";
+import { DimensionSort, SortDirection } from "../../../common/models/sort/sort";
+import { Split, SplitType } from "../../../common/models/split/split";
 import { SPLIT } from "../../config/constants";
 import "../../utils/test-utils";
-import {
-  fillDatasetWithMissingValues,
-  orderByNumberRangeDimensionIncreasing,
-  orderByTimeDimensionDecreasing,
-  orderByTimeDimensionIncreasing,
-  orderByValueDecreasing,
-  orderByValueIncreasing
-} from "./dataset";
+import { fillDatasetWithMissingValues } from "./dataset";
 import { expectedDataset, expectedDatasetReversed, rawDataset, rawDatasetWithTimeDimension, rawDataWithNumberRanges, reversedDatasetWithTimeDimension } from "./test-fixtures";
 
 const timezone = Timezone.UTC;
 
+const mockSplit = (reference: string, type: SplitType, direction: SortDirection) =>
+  new Split({ reference, type, sort: new DimensionSort({ reference, direction }) });
+
 describe("Dataset", () => {
   it("works", () => {
-    expect(fillDatasetWithMissingValues(Dataset.fromJS(rawDataset.data[0][SPLIT] as Dataset), "pv_count", "site", orderByValueDecreasing, timezone).toJS())
+    const siteSplit = mockSplit("site", SplitType.string, SortDirection.descending);
+    expect(fillDatasetWithMissingValues(Dataset.fromJS(rawDataset.data[0][SPLIT] as Dataset), "pv_count", siteSplit, timezone).toJS())
```

Comment:
I'd argue for moving what's inside `expect` clause to a named constant - it's going to be far more readable.